test/sqllogictest/tpch.slt

```diff
@@ -1430,16 +1430,15 @@ ORDER BY
 | |   delta %5 %6.(#0)
 | |   delta %6 %5.(#0)
 | | demand = (#0, #5)
-| Reduce group=(#0, #0)
+| Reduce group=(#0)
 | | agg sum(#5)
-| Filter (#2 > 30000dec)
-| Distinct group=(#0)
 | ArrangeBy (#0)
```

Comment:
I think what has happened here is that we now have the right arrangement for the join, and so we have chosen to use that rather than filter and then arrange.